src/apps/Permissions/Home/Apps.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { CardLayout } from '@aragon/ui'
 import { AppType } from '../../../prop-types'
-import AppCard from '../AppCard'
+import AppCard from '../../../components/AppCard/AppCard'
```

Comment:
Hmm, I think the intention was to move `AppCard` into `src/apps/Permissions/Home/`; do you think we'll use this component elsewhere?

Reply:
I was thinking about App Center yes, but they might be different enough to not share the same component:

<img src=https://user-images.githubusercontent.com/36158/63428386-bdda4100-c417-11e9-916b-85e88dbf7068.png width=200> <img src=https://user-images.githubusercontent.com/36158/63428411-ca5e9980-c417-11e9-9854-972723fad8b8.png width=170>